src/core/runtime/src/add-scp-step.ts

```diff
@@ -103,6 +103,13 @@ export const handler = async (input: AddScpInput) => {
     acceleratorPrefix,
   });
 
+  await scps.attachOrDetachPoliciesToAccounts({
```

Comment:
We are performing add-scps, when an scp is modified in backgroud using cloudwatch event. Need to add same funciton call in 
``` accelerator/src/deployments/runtime/src/ou-validation-events/policy-changes.ts ```

Reply:
I just tested this by editing the SCP attached to a single account (i.e. not attached to any OU's) with a non accelerator role. The change altered one of the statements from "deny" to "allow". In this scenario the SCP policy change was caught and reinstated as it should be. Not sure that the change you mention above is required unless I have miss understood?

Reply:
yeah. You are correct we are already performing policy changes in generic manner.